schema_based_on_diagram.sql

```diff
@@ -0,0 +1,50 @@
+CREATE TABLE patients(
+    id INTEGER NOT NULL,
+    name VARCHAR(255) NOT NULL,
+    date_of_birth DATE NOT NULL,
+    PRIMARY KEY(id)
+);
+
+CREATE TABLE medical_histories(
+    id INTEGER NOT NULL,
+    admitted_at TIMESTAMP(0) WITHOUT TIME ZONE NOT NULL,
+    patient_id INTEGER NOT NULL,
+    status VARCHAR(255) NOT NULL,
+    PRIMARY KEY(id),
+    CONSTRAINT medical_histories_patient_id_foreign FOREIGN KEY(patient_id) REFERENCES patients(id)
+);
+
+CREATE TABLE invoices(
+    id INTEGER NOT NULL,
+    total_amount DECIMAL(8, 2) NOT NULL,
+    generated_at TIMESTAMP(0) WITHOUT TIME ZONE NOT NULL,
+    payed_at TIMESTAMP(0) WITHOUT TIME ZONE NOT NULL,
+    medical_history_id INTEGER NOT NULL,
+    PRIMARY KEY(id),
+    CONSTRAINT invoices_medical_history_id_foreign FOREIGN KEY(medical_history_id) REFERENCES medical_histories(id)
+);
+
+CREATE TABLE treatments(
+id INTEGER NOT NULL,
+type VARCHAR(255) NOT NULL,
+name VARCHAR(255) NOT NULL,
+PRIMARY KEY(id)
+);
+
+CREATE TABLE medical_histories_treatments(
+medical_histories_id INTEGER NOT NULL,
+treatment_id INTEGER NOT NULL,
+PRIMARY KEY(medical_histories_id, treatment_id)
+);
+
+CREATE TABLE invoice_items(
+id INTEGER NOT NULL,
+unit_price DECIMAL(8, 2) NOT NULL,
+quantity INTEGER NOT NULL,
+total_price DECIMAL(8, 2) NOT NULL,
+invoice_id INTEGER NOT NULL,
+treatment_id INTEGER NOT NULL,
+PRIMARY KEY(id),
+CONSTRAINT invoice_items_invoice_id_foreign FOREIGN KEY(invoice_id) REFERENCES invoices(id),
+CONSTRAINT invoice_items_treatment_id_foreign FOREIGN KEY(treatment_id) REFERENCES treatments(id)
+); 
```

Comment:
- [ **OPTIONAL** ] **README.md File:** I think it would be better if you fix the broken `[MIT](./LICENSE)` link in your README file to make it more professional as it shows a **404** error page once clicked. And to fix that, you can only create a new file name `LICENSE` in your root directory and get its content from this **[template](https://github.com/microverseinc/readme-template/blob/master/MIT.md)** or rename this `[MIT](./LICENSE)` to `[MIT](./MIT.md)`, then the created file in your directory name it `MIT.md` and use the same comment in this **[template](https://github.com/microverseinc/readme-template/blob/master/MIT.md)**

Reply:
Thanks @tresorsawasawa